easybuild/easyconfigs/p/PyExtensions/PyExtensions-3.6-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,71 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'Bundle'
+name = 'PyExtensions'
+py_maj_ver = '3'
+py_min_ver = '6'
+py_rev_ver = '1.1'
+pyver = '%s.%s' % (py_maj_ver, py_min_ver)
+version = '%s' % pyver
+
+homepage = 'https://pypi.python.org/pypi'
+description = """This module is a bundle of Python packages on Cray systems based on the module cray-python"""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'pic': True, 'verbose': False}
+
+dependencies = [
+    ('cray-python/%s.%s.%s' % (py_maj_ver, py_min_ver, py_rev_ver), EXTERNAL_MODULE),
+]
+
+# bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+exts_list = [
+    ('Cython', '0.28.2', {
+        'req_py_majver': '3',
+        'req_py_minver': '6',
+        'source_urls': ['https://pypi.python.org/packages/source/c/cython/'],
+    }),
+    ('six', '1.11.0', {
+        'req_py_majver': '3',
+        'req_py_minver': '6',
+        'source_urls': ['https://pypi.python.org/packages/source/s/six/'],
+    }),
+    ('matplotlib', '2.2.2', {
+        'req_py_majver': '3',
+        'req_py_minver': '6',
+        'source_urls': ['https://pypi.python.org/packages/source/m/matplotlib/'],
+    }),
+    ('pandas', '0.22.0', {
+        'req_py_majver': '3',
+        'req_py_minver': '6',
+        'source_urls': ['https://pypi.python.org/packages/source/p/pandas/'],
+    }),
+    ('pygelf', '0.3.1', {
+        'req_py_majver': '3',
+        'req_py_minver': '6',
+        'source_urls': ['https://pypi.python.org/packages/source/p/pygelf/'],
+    }),
+    ('PyMySQL', '0.7.11', {
+        'req_py_majver': '3',
+        'req_py_minver': '6',
+        'source_urls': ['https://pypi.python.org/packages/source/p/pymysql/'],
+    }),
+    ('influxdb', '4.1.1', {
```

Comment:
I do not think we need influx anymore. 
@jgphpc, can you confirm?

Reply:
yes, not needed anymore

Reply:
Thanks @jgphpc and @victorusu, I'll remove the package `influxdb` from the list of extensions